pow/cpuminer/uint256.go

```diff
@@ -24,6 +24,10 @@ import (
 	hsp "github.com/CovenantSQL/HashStablePack/marshalhash"
 )
 
+const (
+	Uint256Size = 8 * 4
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const Uint256Size should have comment (or a comment on this block) or be unexported